std/format/package.d

```diff
@@ -269,119 +269,242 @@ It's also possible to use positional arguments for $(I width), $(I
 precision) and $(I separator) by adding a number and a $(B
 '$(DOLLAR)') after the $(B '*').
 
+$(SECTION4 Types)
+
+The following table contains detailed descriptions for every
+available combination of type and format character. To avoid
+unnecessary repetition, here are some notes that apply to all or
+several entries:
+
+$(UL
+  $(LI If the format character is upper case, the outcome is identical
+       to the lower case version, but uses upper case letters.)
+  $(LI The default precision for floating point numbers is 6 digits.)
+  $(LI Floating point numbers are rounded according to the
+       current rounding mode.)
```

Comment:
What is the current rounding mode?

Reply:
This is decided by the hardware and can be queried and changed (on some computers) by using `std.math.hardware : FloatingPointControl`. If not available, default is round-to-nearest.

Reply:
Suggestion: "Rounding of floating point numbers adheres to the rounding mode of the floating point unit, if available."

Reply:
Yes, that's great.